jsx/Form.js

```diff
@@ -501,6 +501,7 @@ class SelectElement extends Component {
     let multiple = this.props.multiple ? 'multiple' : null;
     let required = this.props.required ? 'required' : null;
     let disabled = this.props.disabled ? 'disabled' : null;
+    let elementClass = this.props.elementClass;
```

Comment:
@HenriRabalais elementClass is already declare at https://github.com/aces/Loris/pull/6862/files#diff-bb24eb9a9e1227db8dd174deb9712093R510